solutions/level1/main.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'cart'
+require_relative 'library'
+require_relative 'book_serializer'
+
+FILE = '../../requirements/level1/data.json'
+
+library = Library.new
+books_hash = JsonReaderWriter.read_file(FILE)
+library.create_books(books_hash)
+
+cart = Cart.new
+cart.calculate_totals(library.books, [1, 2])
```

Comment:
I don't see any place where you added items to your Cart. The OOP approach would be to actually add items to the Cart, something like `Cart.add(Book.find(1))`.

Cart doesn't have multiple totals, so I think this method name is not a good fit. I think it would be recommended to have `cart.total` which knows each time to return the correct result